src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -113,7 +111,7 @@
   [repo-dir path ext]
   (let [relative-path (-> (string/replace path repo-dir "")
                           (string/replace (str "." ext) ""))]
-    (str repo-dir backup-dir "/" relative-path)))
+    (util/node-path.join repo-dir backup-dir relative-path)))
```

Comment:
`node-path.join` cannot handle URLs. IMHO, better add a wrapper for something like "url-safe path join"